addons/website_slides/models/slide_slide.py

```diff
@@ -547,6 +547,10 @@ def action_like(self):
         self.check_access_rule('read')
         return self._action_vote(upvote=True)
 
+    def action_unpublish(self):
```

Comment:
If this method is called only once with a write, just embed it.

Reply:
I m sorry but i don't really understand what you mean by "embed it" ?

Reply:
It is better to embed its code in the caller, don't define a method that does a simple write and is called only once. Otherwise reading is difficult as you have to jump from method to method without reason.

Reply:
Done !